app/src/main/java/com/framgia/soundclound/screen/local/ItemTrackLocalViewModel.java

```diff
@@ -0,0 +1,46 @@
+package com.framgia.soundclound.screen.local;
+
+import android.databinding.BaseObservable;
+import android.view.View;
+
+import com.framgia.soundclound.data.model.Track;
+/**
+ * Created by ADMIN on 1/7/2018.
+ */
+
+public class ItemTrackLocalViewModel extends BaseObservable {
+
+    private Track mTrack;
+    private TrackClickListener mTrackClickListener;
+
+    public ItemTrackLocalViewModel(Track track, TrackClickListener trackClickListener) {
+        mTrack = track;
+        trackClickListener = mTrackClickListener;
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to trackClickListener in new com.framgia.soundclound.screen.local.ItemTrackLocalViewModel(Track, TrackClickListener)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE